OK, this appears to have hit the occasional segfault we've observed before: https://api.travis-ci.org/v3/job/391836746/log.txt

```
[00m   [00;33mrun[00m build/production/tmp/allunittests
[00;31mDebug reporting works: true at ./src/mxnet/Handle.d:465
[16:32:52] src/engine/engine.cc:55: MXNet start using engine: ThreadedEngine
[00mSegmentation fault (core dumped)
submodules/makd/Makd.mak:608: recipe for target '/home/travis/build/sociomantic-tsunami/dmxnet/build/production/tmp/allunittests.stamp' failed
make: *** [/home/travis/build/sociomantic-tsunami/dmxnet/build/production/tmp/allunittests.stamp] Error 139
```

This makes me wonder if there's something about the instantiation of `libmxnet` data structures that we are failing to wait for in our unittests.

--- a/Config.mak
+++ b/Config.mak
@@ -1,5 +1,8 @@
 INTEGRATIONTEST := integrationtest
 
+MXNET_ENGINE_TYPE ?= NaiveEngine
+export MXNET_ENGINE_TYPE
+
 ifeq ($(DVER),2)
 	DC ?= dmd-transitional
 endif

